gulpfile.babel.js

```diff
@@ -0,0 +1,221 @@
+				html: [settings.sources.markup]
+			}))
+			.pipe(gulp.dest(settings.destinations.prod.styles));
+	}
+	return gulp.src(settings.sources.stylesEntryPoint)
+		.pipe(plumber({errorHandler: onError}))
+		.pipe(sourcemaps.init())
+		.pipe(less({
+			plugins: [new Autoprefix(settings.autoPrefix)]
+		}))
+		.pipe(sourcemaps.write('./'))
+		.pipe(gulp.dest(settings.destinations.dev.styles));
+}
+
+export function images() {
+	if (isProdBuild()) {
+		return gulp.src(settings.sources.images)
+			.pipe(imagemin())
+			.pipe(gulp.dest(settings.destinations.prod.images));
+	}
+	return gulp.src(settings.sources.images)
+		.pipe(gulp.dest(settings.destinations.dev.images));
+}
+
+export function clientScripts() {
+	if (isProdBuild()) {
+		return gulp.src(settings.sources.scripts)
+			.pipe(rollup({
+				entry: './src/app/index.js'
+			}))
+			.pipe(babel())
+			.pipe(uglify())
+			.pipe(rename({
+				baseName: 'client',
+				suffix: '.min'
+			}))
+			.pipe(gulp.dest(settings.destinations.prod.scripts));
+	}
+	return gulp.src(settings.sources.scripts)
+		.pipe(rollup({
+			entry: './src/app/index.js'
+		}))
+		.pipe(babel())
+		.pipe(gulp.dest(settings.destinations.dev.scripts));
+}
+
+export function vendorScripts() {
+	const b = browserify({...browserifyInc.args});
+	settings.sources.externalJs.forEach(dep => b.require(dep));
+	browserifyInc(b, {cacheFile: './.browserify-cache.json'});
+
+	if (isProdBuild()) {
+		return b.bundle()
+			.pipe(source('vendor.min.js'))
+			.pipe(buffer())
+			.pipe(uglify())
+			.pipe(gulp.dest(settings.destinations.prod.libs));
+	}
+	return b.bundle()
+		.pipe(source('vendor.js'))
+		.pipe(buffer())
+		.pipe(gulp.dest(settings.destinations.dev.libs));
+}
+
+export function bundleExternalCSS(done) {
+	const files = pkgJson.bootstrapKickstart.bundleCSS.map(sourcePath => path.join('node_modules/', sourcePath));
+	if (!files.length) return done();
+	if (isProdBuild()) {
+
+		return gulp.src(files)
+			.pipe(cleanCss())
+			.pipe(concat('libs.min.css'))
+			.pipe(gulp.dest(settings.destinations.prod.libs));
+	}
+	return gulp.src(files)
+		.pipe(concat('libs.css'))
+		.pipe(gulp.dest(settings.destinations.dev.libs));
+}
+
+export function copyStaticFiles() {
+	return gulp.src(settings.sources.staticFiles.map(sourcePath => path.join('node_modules/', sourcePath)), {base: 'node_modules/'})
+		.pipe(gulp.dest(isProdBuild() ? settings.destinations.prod.libs : settings.destinations.dev.libs));
+}
+
+export function lint() {
+	if (isProdBuild()) {
+		return gulp.src([...settings.sources.scripts, './*.js'])
+			.pipe(eslint())
+			.pipe(eslint.format())
+			.pipe(eslint.failAfterError());
+	}
+	return gulp.src([...settings.sources.scripts, './*.js'])
+		.pipe(eslint())
+		.pipe(eslint.format())
+		.pipe(eslint.failAfterError())
+		.on('error', onError);
+}
+
+export function security(done) {
+	if (isProdBuild()) {
+		nsp({package: path.join(__dirname, '/package.json')}, done);
+	} else {
+		done();
+	}
+}
+
+export function html() {
```

Comment:
@nicolaisueper @DanielBrommer 
I think this task should be renamed to `processHtml`.